test/Project.toml

```diff
@@ -1,3 +1,7 @@
 [deps]
+AbaqusReader = "bc6b9049-e460-56d6-94b4-a597b2c0390d"
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
```

Comment:
You should add compat entries for those two and not wait for CompatHelper to kick in